src/Stratis.Bitcoin/Connection/ConnectionManager.cs

```diff
@@ -89,6 +91,10 @@ public IReadOnlyNetworkPeerCollection ConnectedPeers
 
         private readonly PayloadProvider payloadProvider;
 
+        private readonly ISignals signals;
+
+        public IEnumerable<PeerConnectionModel> peerConnectionModels { get; set; }
```

Comment:
You don't need to store a class variable for this. Just create the list in the method an dpublish it 👍